km_api/know_me/serializers/emergency_contact_serializers.py

```diff
@@ -0,0 +1,32 @@
+"""Serializers for the ``EmergencyContact`` model.
+"""
+
+from rest_framework import serializers
+
+from know_me import models
+
+
+class EmergencyContactListSerializer(serializers.HyperlinkedModelSerializer):
+    """
+    Serializer for ``EmergencyContact`` instances.
+    """
+    url = serializers.HyperlinkedIdentityField(
+        view_name='know-me:emergency-contact-detail')
+
+    class Meta:
+        fields = ('id', 'url', 'name', 'relation', 'phone_number',
+                  'alt_phone_number', 'email')
+        model = models.EmergencyContact
+
+
+class EmergencyContactDetailSerializer(EmergencyContactListSerializer):
```

Comment:
What's the point of having the separate list and detail serializers if they have the same fields? If they contain the same info, I would rather have a sing `EmergencyContactSerializer`.